docs/root/intro/arch_overview/upstream/load_reporting_service.rst

```diff
@@ -0,0 +1,39 @@
+.. _arch_overview_load_reporting_service:
+
+Load Reporting Service (LRS)
+============================
+
+The Load Reporting Service provides a mechanism by which Envoy can emit Load Reports to a management
+server at a regular cadence. Below is the envoy config to enable Load reporting service.
+
+This will initiate a bi-directional stream with a management server. Upon connecting, the management
+server can send a :ref:`LoadStatsResponse <envoy_v3_api_msg_service.load_stats.v3.LoadStatsResponse>`
+to a node it is interested in getting the load reports for. Envoy in this node will start sending
+:ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`. This is done periodically
+based on the :ref:`load reporting interval <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>`
+
+For details, take a look at the `sandbox example <https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/load_reporting_service.html>`_.
+
+.. code-block:: yaml
```

Comment:
This can get out of date. I would recommend just `:repo:` linking to the config file in the example.